internal/host/transition_test.go

```diff
@@ -4266,6 +4266,62 @@ var _ = Describe("Refresh Host", func() {
 			})
 		}
 	})
+	Context("Platform validations", func() {
+
+		BeforeEach(func() {
+			mockDefaultClusterHostRequirements(mockHwValidator)
+			mockEvents.EXPECT().AddEvent(gomock.Any(), infraEnvId,
+				gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).
+				AnyTimes()
+		})
+
+		tests := []struct {
+			name                  string
+			hostPlatform          string
+			userManagedNetworking bool
+			dstState              string
+		}{
+			{
+				name:                  "validate platform OpenStack and userManagedNetwork false",
+				hostPlatform:          "OpenStack Compute",
```

Comment:
maybe it should be a const that will be used in `invalidPlatforms` as well? it will be easier to find references this way.

Reply:
Good call.  Can follow the vsphere code in internal/common/platform.go